streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java

```diff
@@ -58,6 +62,13 @@
     @Deprecated
     public final InternalTopologyBuilder internalTopologyBuilder = new InternalTopologyBuilder();
 
+    private Topology.AutoOffsetReset translateAutoOffsetReset(final TopologyBuilder.AutoOffsetReset resetPolicy) {
```

Comment:
I think if we deprecate a class we do not need to deprecate all its member functions, but we need to check any callers that may pass in this class object or return this class object as deprecated. Reference:

https://stackoverflow.com/questions/15908887/how-to-annotate-deprecation-of-a-class-in-java